sdk/storage/azure-storage-blob/azure/storage/blob/_blob_client.py

```diff
@@ -527,7 +527,7 @@ def _download_blob_options(self, offset=None, length=None, **kwargs):
 
     @distributed_trace
     def download_blob(self, offset=None, length=None, **kwargs):
-        # type: (Optional[int], Optional[int], **Any) -> Iterable[bytes]
+        # type: (Optional[int], Optional[int], **Any) -> StorageStreamDownloader
```

Comment:
This comment is for the doc you didn't edit ^_^
StorageStreamDownloader is not a iterable generator